controllers/ListItemController.php

```diff
@@ -17,6 +17,34 @@
 
 	class ListItemController
 	{
+		/**
+			* Swagger doc block
+			*
+			* @OA\Get(
+			* path="ListItem/get_list_items",
+			* summary="Find list items by list id",
+			* @OA\Parameter(
+			*		name="listID",
+			* 	in= "query",

```

Comment:
Here a more apropriate option would be to use `path` instead of the `query` as `query` refers to arguments passed down using `?listID` or `&listID` where as path means it like `/list/item/12`  where 12 is the `listID `

Reply:
made the required change.
Is there any way I can know which parameters are in query and which are path? Is there any file which mentions this or is it something evident from the code which I am not aware of?

Reply:
In the [routes folder](https://github.com/gigili/did-you-buy-it-api/tree/main/routes) there are route files which have all the routes listed, and from there you can see which ones take arguments as part of the path (ex /api/user/{userID})